client/my-sites/plans/jetpack-plans/product-store/hero-image/index.tsx

```diff
@@ -47,17 +47,29 @@ const HERO_IMAGES: Record< string, { lg: string; md: string; sm: string } > = {
 	},
 };
 
-export const HeroImage: React.FC< HeroImageProps > = ( { item } ) => {
-	const smallScreenImage = HERO_IMAGES[ item.productSlug ]?.sm;
-	const largeScreenImage = HERO_IMAGES[ item.productSlug ]?.lg;
-
-	//mediumScreenImage is just only for Jetpack Complete, for other products it is same as largeScreenImage
-	const mediumScreenImage = HERO_IMAGES[ item.productSlug ]?.md ?? largeScreenImage;
-
-	if ( ! smallScreenImage || ! mediumScreenImage || ! largeScreenImage ) {
-		return null;
-	}
+const HERO_IMAGES_API_FAMILY: Record< string, { lg: string; md: string; sm: string } > = {
+	[ 'jetpack-security-t1' ]: {
+		lg: SecurityLg,
+		md: SecurityLg,
```

Comment:
For future reference: this is not a typo; there is no medium image at the moment. Ideally we'd note this as a comment in code.